arch/arm/src/sama5/Kconfig

```diff
@@ -3615,6 +3615,16 @@ config SAMA5_ADC_SWTRIG
 	---help---
 		A-to-D Conversion is initiated only by software via an ioctl()
 
+config SAMA5_ADC_PERIODIC_TRIG
+	bool "Periodic trigger"
+	---help---
+		A-to-D Conversion is triggered periodically
+
+config SAMA5_ADC_CONTINUOUS_TRIG
```

Comment:
No - the modes are different. 

Periodic means the ADC is triggered regularly at a rate set in the Trigger register, using the value (now) set in Kconfig via SAMA5_ADC_TRIGGER_PERIOD.

Continuous means it will start a new conversion sequence as soon as the current one finishes.

Reply:
If so you should use choice/endchoice like this:
https://github.com/apache/incubator-nuttx/blob/master/Kconfig#L89-L105
to avoid the user select both options.

Reply:
OK...may be a quirk of Kconfig but the selections actually work as being mutually exclusive

Reply:
Think I may have included an incorrect Kconfig - I'll check.